### PR TITLE
Update ECS Submodule to support separate Portal Deployment

### PR DESCRIPTION
**Pull request by mattholmes15**

### CODE REVIEW

File: examples/hybrid_ecs/main.tf [hunk 1]
```diff
@@ -261,6 +261,83 @@ module "create_kong_cp" {
...
+
+  skip_rds_creation = true
+  template_file     = "control_plane"
+
+  environment = local.environment
+  service     = var.service
+  description = var.description
+  tags        = var.tags
+
+  depends_on = [module.create_kong_cp]
```
Comment (dan-hill2802) [inline examples/hybrid_ecs/main.tf:338]: I don't think this is needed, you already have the relationship set on lines 323 and 324

File: examples/hybrid_ecs/main.tf [hunk 2]
```diff
@@ -261,6 +261,83 @@ module "create_kong_cp" {
...
+
+  private_subnets    = module.create_kong_cp.private_subnet_ids
+  availability_zones = module.create_kong_cp.private_subnet_azs
+
+  image_url = var.image_url
+
+  ecs_target_group_arns = local.target_group_portal
+
+  skip_rds_creation = true
+  template_file     = "control_plane"
```
Comment (dan-hill2802) [inline examples/hybrid_ecs/main.tf:331]: this variable isn't ultimately used, so need removing from the codebase

File: main.tf [hunk 3]
```diff
@@ -8,6 +8,11 @@ locals {
       "telemetry"  = 8006,
       "status"     = 8100
     }
+    portal = {
+      "portal_gui" = 8446,
+      "portal_api" = 8447,
```
Comment (dan-hill2802) [inline main.tf:13]: these are HTTPS ports, but the control plane above is using HTTP ports. Why the difference?

File: variables.tf [hunk 4]
```diff
@@ -816,3 +816,21 @@ variable "kong_portal_enabled" {
...
+  description = "(Optional) The protocol used for the portal GUI."
+  type        = string
+  default     = ""
+}
+
+variable "kong_portal_api_url" {
+  description = "(Optional) The Portal API URL of the Portal."
+  type        = string
+  default     = ""
+}
```
Comment (dan-hill2802) [inline variables.tf:836]: these have an overlap with `var.kong_ssl_uris` (line 250). Can we not use the existing var instead?

File: main.tf (hunk 3 above)
Comment (mattholmes15) [reply main.tf:13]: Which ports are you looking at? 8444/8445 are HTTPS, 8005 and 8006 are used for websockets

File: variables.tf (hunk 4 above)
Comment (mattholmes15) [reply variables.tf:830]: As discussed, `kong_ssl_uris` will be deprecated in a future version

File: examples/hybrid_ecs/main.tf (hunk 1 above)
Comment (mattholmes15) [reply examples/hybrid_ecs/main.tf]: Removed

File: examples/hybrid_ecs/main.tf (hunk 2 above)
Comment (mattholmes15) [reply examples/hybrid_ecs/main.tf]: Removed

**Review by alv-lop — APPROVED** [review]

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**